### PR TITLE
resolve framework config before Processor::processConfiguration()

### PR DESCRIPTION
**Pull request by thezapp01**
The configuration may contain links to data from the environment. What can cause errors in
Symfony\Component\Config\Definition\Processor::processConfiguration()

### CODE REVIEW

File: lib/DependencyInjection/Compiler/SerializerContextConfigurationPass.php [hunk 1]
```diff
@@ -18,7 +18,10 @@ public function process(ContainerBuilder $container): void
         if ($container->hasExtension('framework')) {
             /** @var FrameworkExtension $framework */
             $framework = $container->getExtension('framework');
+            $resolvingBag = $container->getParameterBag();
+
             $frameworkConfigs = $container->getExtensionConfig('framework');
+            $frameworkConfigs = $resolvingBag->resolveValue($frameworkConfigs);
```
Comment (alekitto) [inline lib/DependencyInjection/Compiler/SerializerContextConfigurationPass.php]: Probably `$container->resolveEnvPlaceholders` should be used here, as used in CheckDependencyPass.php:L20

**Review by alekitto — CHANGES_REQUESTED** [review]
Hi @thezapp01,
👍 for the fix.

I suggested another call instead of `ParameterBag::resolveValue` because is already used in another compiler pass.
Additionally, can you please rebase upon the latest `master` commit?

File: lib/DependencyInjection/Compiler/SerializerContextConfigurationPass.php (hunk 1 above)
Comment (thezapp01) [reply lib/DependencyInjection/Compiler/SerializerContextConfigurationPass.php]: Hi @alekitto,  
I replaced with $container->resolveEnvPlaceholders and rebase upon the latest master commit

Comment (alekitto) [reply lib/DependencyInjection/Compiler/SerializerContextConfigurationPass.php]: I still see a merge commit, probably something went wrong with the rebase.

**Comment by alekitto** [comment]
Merged in https://github.com/alekitto/messenger-extra/commit/8c8570ba5c690a68c682b3565e5b645007f0f36d

Thank you @thezapp01

**Pull request closed without merging**